content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -589,11 +594,9 @@ SPECIAL_TYPE
 If there are some files in the `/etc/config/` directory, they will be deleted.
 {{< /caution >}}
 
-{{< note >}}
-Text data is exposed as files using the UTF-8 character encoding. To use some other character encoding, use binaryData.
-{{< /note >}}
+Text data is exposed as files using the UTF-8 character encoding. To use some other character encoding, use `binaryData`.
 
-### Add ConfigMap data to a specific path in the Volume
+### Add ConfigMap data to a specific path in the volume
```

Comment:
This is referring to the `path` field in the Volume configuration?

Reply:
The way I understand it, it's about telling the kubelet to write data into a mounted filesystem based on what's in a given ConfigMap.

Reply:
OK